main.cpp

```diff
@@ -104,6 +104,11 @@ int main(int argc, char *argv[])
     app.setOrganizationDomain("getmonero.org");
     app.setOrganizationName("monero-project");
 
+    QString configPath;
+    QSettings *settings = new QSettings(QSettings::NativeFormat, QSettings::UserScope, app.organizationName(), app.applicationName());
```

Comment:
AFAIK there is no need to do dynamic memory allocation here. 

`QSettings settings(QSettings::NativeFormat, QSettings::UserScope, app.organizationName(), app.applicationName());`

should work

Reply:
@pazos We can't switch to `QSettings::IniFormat` without some sort of migration though.

On Windows, registry->.ini
On OSX, plist->.ini (?)
On Linux, .conf->.ini

Would say that's out of this scope for this PR.

Reply:
linux stores settings in ini, it doesn't have its own native format.

migration is ok, but can lead to bugs and add ugly code. IMO is better to do the switch in a major release and never look back 😄